### PR TITLE
added terraform modules

### PR DESCRIPTION
**Pull request by geoffrey1330**

### CODE REVIEW

File: modules/vpc/main.tf [hunk 1]
```diff
@@ -0,0 +1,109 @@
+resource "aws_vpc" "main" {
+  cidr_block = "10.0.0.0/16"
+}
+
+locals {
+  public_cidr = ["10.0.0.0/24", "10.0.1.0/24"]
+
+  private_cidr = ["10.0.2.0/24", "10.0.3.0/24"]
+}
```
Comment (vladimirmukhin) [inline modules/vpc/main.tf]: Please have all cidr_blocks as variables

File: modules/ec2/vars.tf [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  type = string
+}
+
+variable "public-subnet_id" {
+
+}
+
+variable "private-subnet_id" {
+
+}
```
Comment (vladimirmukhin) [inline modules/ec2/vars.tf]:
```suggestion
variable "public-subnet_id" {}

variable "private-subnet_id" {}
```

File: modules/vpc/vars.tf [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+#   type = string
+# }
+
+# variable "public-subnet_id" {
+
+# }
+
+# variable "private-subnet_id" {
+
+# }
```
Comment (vladimirmukhin) [inline modules/vpc/vars.tf]: Please don't leave commented-out code

File: level2/data.tf [hunk 4]
```diff
@@ -7,4 +7,4 @@ data "terraform_remote_state" "networking" {
     region         = "us-east-1"
     dynamodb_table = "israel-dynamo-terraform"
   }
-}
+}
```
Comment (vladimirmukhin) [inline level2/data.tf]: Please add trailing newline

File: level2/main.tf [hunk 5]
```diff
@@ -11,49 +11,11 @@ provider "aws" {
...
-  user_data = <<EOF
-    #!/usr/bin/env bash
-    sudo yum update -y
-    sudo yum install -y httpd
-    sudo systemctl start httpd && sudo systemctl enable httpd
-    EOF
-}
+  public-subnet_id = data.terraform_remote_state.networking.outputs.public-subnet_id
+  private-subnet_id = data.terraform_remote_state.networking.outputs.private-subnet_id
+}
```
Comment (vladimirmukhin) [inline level2/main.tf]: Please add trailing newline

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

File: modules/vpc/main.tf (hunk 1 above)
Comment (geoffrey1330) [reply modules/vpc/main.tf]: OK

**Review by vladimirmukhin — APPROVED** [review]

**Pull request merged**